### ISSUE TITLE
compound_search_operator argument missing from data.okta_users

### ISSUE THREAD

**Issue by dli-spoton** [okta/terraform-provider-okta#1077]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v1.0.4
on darwin_amd64
+ provider registry.terraform.io/okta/okta v3.25.0
```

### Affected Resource(s)

- data.okta_users

### Terraform Configuration Files

```hcl
data "okta_users" "example" {
  search {
    name       = "status"
    value      = "DECOMISSIONED"
    comparison = "eq"
  }
  search {
    name       = "status"
    value      = "STAGED"
    comparison = "eq"
  }
  compound_search_operator = "or"
}
```

### Expected Behavior

Join the search blocks with an `or` operator

### Actual Behavior

```terraform
│ Error: Unsupported argument
│ 
│   on main.tf line 29, in data "okta_users" "example":
│   29:   compound_search_operator = "or"
│ 
│ An argument named "compound_search_operator" is not expected here.
```

### Steps to Reproduce

1. `terraform plan`

### References

- #1027

**Comment by monde** [comment]
@dli-spoton we just did a release yesterday, I'll try and do a quick bug fix today.

**Comment by monde** [comment]
I have a fix queued up in PR #1081 and am waiting for a colleague to review it before releasing.

**Comment by monde** [comment]
v3.25.1 is available in the terraform registry
